translate 00_introduction.js

ibufu opened the pull request:
we were just calling them differently. 
这句话不确定这里的call是调用还是称呼的意思

ibufu committed:
translate 00_introduction.js

dorayx commented:
> So is flux just a matter of new **vocabulary**? Not exactly. But vocabulary DOES matter, because by introducing these new terms we are now able to **express more precisely things** that were regrouped under various terminologies... For example ... we were just **calling** them differently. 

结合上下文，我觉得这个 **calling** 应该是 **express** 的意思，也就是称呼吧。

camsong commented on 00_introduction.js:
```diff
@@ -1,18 +1,18 @@
+// 并且努力去把它们和那些理论性的概念（action/actions creator、store、dispatcher 等等）给联系起来。
+// 等我用了 Redux 之后，我才发现原来flux比我想象的要简单很多。
+// 这些都归功于 Redux 良好的设计和移除了许多其它框架引入的“反样板代码功能（anti-boilerplate features）”。
+// 我现在觉得用 Redux 来学习 flux 比用其他框架来学习要很好多。
```
这些都归功于 Redux 通过优良的设计来减少样板文件，而其它框架则是为了减少样板文件却又引入了很多新的代码。
我现在觉得用通过 Redux 来学习 flux 比通过其他框架高效得多。

camsong commented on 00_introduction.js:
```diff
@@ -35,57 +35,57 @@
-// applications, we were just calling them differently. And instead of having handlers for those
-// actions directly modify Models or Views, flux ensures all actions go first through something called
-// a dispatcher, then through our stores, and finally all watchers of stores are notified.
+// 所以，flux 就只是一个新词汇表么？不全是，但是词汇表是很重要的，
```
『新词汇表』改为『新名词』更合适吧

ibufu replied:
因为上文提及了Flux 里的 Store Action Dispatch 等名词，所以我感觉这里的 Flux像多个名词的集合。

camsong commented on 00_introduction.js:
```diff
@@ -35,57 +35,57 @@
-// views can finally update. So in the end, data always flows in one way:
+// 来看一下我们是如何避免 Store A 和 Store B 直接相关联的。 
+// 每一个 Store 只能被一个 action 修改，别无他选。
+// 并且当所有 Store 回应了 action 后，最终所有 View 都会更新。由此可见，数据总是以一种方式进行流动：
```
数据总是以一种方式进行流动 => 数据总是沿一个方向流动：

ibufu committed:
Update the translation:
 1.translate 'vocabulary' to '词汇';
 2.translate 'thanks to Redux being very well designed and having removed a lot of "anti-boilerplate features" introduced by other frameworks I tried before. I now feel that Redux is a much better way to learn about flux' to '这些都归功于 Redux 通过优良的设计来减少样板文件，而其它框架则是为了减少样板文件却又引入了很多新的代码。我现在觉得用通过 Redux 来学习 flux 比通过其他框架高效得多。';
 3.translate 'data always flows in one way' to '数据总是沿一个方向流动'.

pandazki commented on 00_introduction.js:
```diff
@@ -1,18 +1,18 @@
-// by other frameworks I tried before. I now feel that Redux is a much better way to learn about flux
-// than many other frameworks. That's why I want now to share with everyone, using my own words,
-// flux concepts that I am starting to grasp, focusing on the use of Redux.
+// 为什么要有这个教程呢？
```
// 为什么会有这个教程呢？
// 在我尝试学习 Redux 的时候，我发现之前的阅读过的一些文章加上个人的经验，让我对 flux 产生了一些错误的认知。
// 当然，我不是说那些关于 flux 的文章写得不好，只是我没能正确地领会其中的概念。
// 到头来，我只是对着各种 flux 框架（Reflux、Flummox、FB Flux）的文档照猫画虎，并试着把它们和之前了解到的理论概念联系起来。

我觉得 introduction 里面的前几句话对初学者来说比较重要，完全的直译会有一种莫名的艰深感，其实前面就是些客套话。
我试着修改了一点，供参考。

camsong replied:
:+1: 译的像讲故事，流畅比保留原义更重要

ibufu replied:
👍👍👍👍👍👍

ibufu committed:
Update the translation of 00_introduction.
